examples/metrics/src/main/java/io/opentelemetry/example/metrics/LongValueObserverExample.java

```diff
@@ -21,9 +21,9 @@ public static void main(String[] args) {
         .build();
 
     observer.setCallback(
-        new LongValueObserver.Callback<LongValueObserver.ResultLongValueObserver>() {
+        new LongValueObserver.Callback<LongResult>() {
```

Comment:
How did this end up here? ;)

Reply:
I have the examples project wired up in my IDE, so I have to fix failures to be able to compile. I can move this to a separate PR, if you prefer.

Reply:
removed. will submit separately